src/renderer/coremods/commands/index.ts

```diff
@@ -9,35 +11,79 @@ import { loadCommands, unloadCommands } from "./commands";
 const logger = Logger.api("Commands");
 const injector = new Injector();
 
-interface ApplicationCommandSearchStoreMod {
-  useDiscoveryState: (...args: unknown[]) =>
+type commandState =
```

Comment:
```suggestion
type CommandState =
```
A general convention in TypeScript is to use PascalCase for types/interfaces. Also replace the name below.